### PR TITLE
Kubernetes 1.3

### PR DESCRIPTION
**Pull request by macat**

### CODE REVIEW

File: pkg/deployment/kubernetes.go [hunk 1]
```diff
@@ -110,15 +110,6 @@ func (d *KubernetesDeployment) steps() ([]TaskStep, error) {
 	var steps = []TaskStep{}
 	for _, task := range d.Playbook.Tasks {
 		if task.PodManifest != "" {
```
Comment (poopoothegorilla) [inline pkg/deployment/kubernetes.go]: wait what happens now?

File: pkg/deployment/pod_manifest_step_test.go [hunk 2]
```diff
@@ -1,113 +0,0 @@
...
-				}
-				w.Modify(pod)
-				pod.Status = v1.PodStatus{
-					Phase: v1.PodRunning,
-				}
-				w.Modify(pod)
-				pod.Status = v1.PodStatus{
-					Phase: v1.PodSucceeded,
-				}
-				w.Modify(pod)
```
Comment (poopoothegorilla) [inline pkg/deployment/pod_manifest_step_test.go]: :*( tear

**Comment by poopoothegorilla** [comment]
+1

File: pkg/deployment/kubernetes.go (hunk 1 above)
Comment (macat) [inline pkg/deployment/kubernetes.go]: Nothing, I want to ditch pod manifests. I'll do it in a separate PR.

Comment (therealplato) [inline pkg/deployment/kubernetes.go]: @macat want to change this to`if task.PodManifes == ""` to eliminate the `else`?

**Comment by therealplato** [comment]
+1

File: pkg/deployment/kubernetes.go (hunk 1 above)
Comment (macat) [inline pkg/deployment/kubernetes.go]: yup.

Comment (macat) [inline pkg/deployment/kubernetes.go]: well, actually nope, I have another branch removing this functionality.

**Pull request merged**